src/Models/TableBlock.php

```diff
@@ -34,11 +34,20 @@ class TableBlock extends BaseElement
 
     private static $icon = 'font-icon-block-table-data';
 
+    /**
+     * Whether to automatically augment the links with Web Government Standards compliant attributes and markup.
+     * To support CWP module which comes with the RichLinksExtension.
+     * @config
+     * @var string
+     */
+    private static $enable_rich_links = false;
+
     private static $db = [
         'TableDescription' => 'HTMLText',
         'TableCaption' => 'Varchar',
         'NumberOfColumns' => 'Int',
         'FirstRowIsHeader' => 'Boolean',
+        'BoldHeader' => 'Boolean',
```

Comment:
Other than adding an option in the CMS admin, `BoldHeader` doesn't seem to be used for anything.